### DISCUSSION TITLE
Create a GeoBlacklight Roadmap

### DISCUSSION THREAD

**Discussion by karenmajewicz** [geoblacklight/geoblacklight#1261] [General]

**Comment by karenmajewicz** [comment]
Develop a long range roadmap leading to GeoBlacklight 3.0

**Comment by karenmajewicz** [comment]
Make that 4.0

**Comment by lenathelibrarian** [comment]
[Draft](https://docs.google.com/document/d/1W7d5SBsX02XYt7jHrM29IYK5m0cHoPfpD8_vxC3dHYQ/edit?usp=sharing)
